backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/inventory/interceptor/M_Inventory.java

```diff
@@ -83,11 +88,26 @@ public void beforeComplete(final I_M_Inventory inventoryRecord)
 		inventoryLineRecordService.syncToHUs(inventoryRecord);
 	}
 
+	@DocValidate(timings = ModelValidator.TIMING_BEFORE_REVERSECORRECT)
+	public void checkHUTransformationBeforeReverseCorrect(final I_M_Inventory inventory)
+	{
+		final InventoryId inventoryId = InventoryId.ofRepoId(inventory.getM_Inventory_ID());
+		final ImmutableMap<InventoryLineId, HuId> inventoryHuIds =
+				inventoryDAO.retrieveLinesForInventoryId(inventoryId, I_M_InventoryLine.class)
```

Comment:
M_InventoryLines lines might *or* might not directly reference an M_HU.
One line might have many HUs (via `M_InventoryLine_HU`).

As written today:
> that depends. but i think the repository method will give you the inventory's HUs, no matter if they are linked via M_InventoryLine_HU or M_InventoryLine

please don't ignore such remarks, even if i might be wrong and the repository method might not work as i wrote.
(i think that the code i had in mind is here: `de.metas.handlingunits.inventory.InventoryRepository#toInventoryLine(de.metas.handlingunits.model.I_M_InventoryLine, java.util.Collection<de.metas.handlingunits.model.I_M_InventoryLine_HU>)`)